CRM/Core/BAO/CustomField.php

```diff
@@ -129,6 +129,7 @@ public static function dataToHtml() {
    * @return CRM_Core_DAO_CustomField
    */
   public static function create(&$params) {
+
```

Comment:
Code formatting changes should probably go in a separate PR to clarify the actual relevant changes.

I recommend using `git add --patch` to review that changes proposed are relevant before committing.